### PR TITLE
UI: Update Sidebar UI

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description
Update to sidebar UI

About # (link your issue here)
#291

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/components/Link.tsx [hunk 1]
```diff
@@ -26,12 +29,19 @@ export default Link
...
     href = '#',
     children,
-    className = 'px-4 py-2 w-52 rounded-3xl line md:p-1 xl:p-3 whitespace-nowrap',
+    className = 'pt-2 pr-2 w-52 rounded-xl line xl:pr-3 whitespace-nowrap',
+    onPress,
     ...rest
-}: React.PropsWithoutRef<ReactRouterNavLinkProps<S>> & React.RefAttributes<HTMLAnchorElement>): JSX.Element {
+}: React.PropsWithoutRef<ReactRouterNavLinkProps<S>> &
+    React.RefAttributes<HTMLAnchorElement> & { onPress: typeof noop }): JSX.Element {
+    const textColor = useColorModeValue('goodGrey.700', 'goodGrey.300')
```
Comment (johnsmith-gooddollar) [inline src/components/Link.tsx:37]:
```suggestion
    const textColor = useColorModeValue('goodGrey.700', 'goodGrey.300')
    
```

File: src/components/WalletBalance/index.tsx [hunk 2]
```diff
@@ -12,28 +13,33 @@ export type WalletBalanceProps = {
 
 const chains = Object.values(AdditionalChainId)
 
-const WalletBalance = memo(({ balances, chainId }: WalletBalanceProps): JSX.Element | null => (
-    <div className="flex flex-col">
-        {balances &&
-            Object.entries(balances).map((balance, index) => {
-                const amount = balance[1]
+const WalletBalance = memo(({ balances, chainId }: WalletBalanceProps): JSX.Element | null => {
+    const textColor = useColorModeValue('goodGrey.700', 'goodGrey.300')
```
Comment (johnsmith-gooddollar) [inline src/components/WalletBalance/index.tsx:17]:
```suggestion
    const textColor = useColorModeValue('goodGrey.700', 'goodGrey.300')
    
```

File: src/components/WalletBalance/index.tsx [hunk 3]
```diff
@@ -12,28 +13,33 @@ export type WalletBalanceProps = {
...
-                                amount?.format({ suffix: '', prefix: amount.currency?.ticker + ' - ' })
-                            )}
-                        </span>
-                    </Fragment>
-                )
-            })}
-    </div>
-))
+                    if (balance[0] === 'GDX' && chains.includes(chainId as any)) {
+                        return <div key={index}></div>
```
Comment (johnsmith-gooddollar) [inline src/components/WalletBalance/index.tsx:25]: again index as key. use ticker (GD, GDX etc) instead

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

File: src/components/WalletBalance/index.tsx (hunk 3 above)
Comment (L03TJ3) [reply src/components/WalletBalance/index.tsx]: done

**Pull request merged**